mint.json

```diff
@@ -102,6 +102,7 @@
           "pages": [
             "send-data/cloudfront",
             "send-data/cloudwatch",
+            "send-data/aws-dot",
             "send-data/aws-firelens",
             "send-data/aws-iot-rules",
             "send-data/aws-s3",
```

Comment:
not related to this PR but noticed inconsistency. This is called `aws-s3` but `cloudwatch` is without `aws-` prefix.

nit pick: its usually called [Amazon S3](https://aws.amazon.com/pm/serv-s3/?gclid=Cj0KCQjw8MG1BhCoARIsAHxSiQkn-ZvYD22N6MboXpmDbko_rHPvZebjoLjNze8IsMluCO45XUoJAnEaAmcSEALw_wcB&trk=3cda67b9-5fb7-4d3f-84e8-40b544661f21&sc_channel=ps&ef_id=Cj0KCQjw8MG1BhCoARIsAHxSiQkn-ZvYD22N6MboXpmDbko_rHPvZebjoLjNze8IsMluCO45XUoJAnEaAmcSEALw_wcB:G:s&s_kwcid=AL!4422!3!645208988806!e!!g!!amazon%20s3!19580264380!143903638703). Not sure though why AWS uses Amazon instead of AWS sometimes.

Reply:
Well spotted! It would be good to have consistent naming in the file names, but I don't think it's worth changing the URL and add redirects. I'll pay attention to this in the text, though.